packages/flutter/test/widgets/flow_test.dart

```diff
@@ -28,6 +29,22 @@ class TestFlowDelegate extends FlowDelegate {
   bool shouldRepaint(TestFlowDelegate oldDelegate) => startOffset == oldDelegate.startOffset;
 }
 
+class OpacityFlowDelegate extends FlowDelegate {
+  OpacityFlowDelegate(this.opacity);
+
+  double opacity;
+
+  @override
+  void paintChildren(FlowPaintingContext context) {
+    for (int i = 0; i < context.childCount; ++i) {
+      context.paintChild(i, opacity: opacity);
+    }
+  }
+
+  @override
+  bool shouldRepaint(TestFlowDelegate oldDelegate) => true;
```

Comment:
technically (just in case anyone copies and pastes this code, it doesn't matter for your purposes) this should be `opacity != oldDelegate.opacity`.

Reply:
done